CITATION

```diff
@@ -1,6 +1,6 @@
 If you use Astropy for work/research presented in a publication (whether
-directly, or as a dependency to another package), please include the following
-acknowledgment:
+directly, or as a dependency to another package), we recommend and encourage 
+(but do not explicitly require) the following acknowledgment:
```

Comment:
I'm :-1: on the "(but do not explicitly require)", and :+1: on "we recommend and encourage".  I feel like the parenthetical part is a little too...I'm not sure how to put it...but "weak".  I don't want to give the impression that it's perfectly fine not to cite/acknowledge, because that is not the case.  It is strongly encouraged.  

The misunderstanding about matplotlib was so strange it seemed almost willful.  I don't think "we recommend and encourage" gives any impression that there's a legal obligation to, and is better than the existing wording.